nimbus.fml.yaml

```diff
@@ -19,7 +19,7 @@ features:
             "recently-saved": true,
             "recent-explorations": true,
             "pocket": true,
-            "contile-top-sites": false,
+            "contile-top-sites": true,
```

Comment:
If the experiment ended could we just remove it from the app?
And then have the `showContileFeature` setting just become
```xml
    var showContileFeature by booleanPreference(
        key = appContext.getPreferenceKey(R.string.pref_key_enable_contile),
        default = true,
    )
```

Reply:
Looks like this was the experiment for contile https://experimenter.services.mozilla.com/nimbus/firefox-android-sponsored-shortcuts-experiment with @lwa-moz being the owner.
Asking Loren if we need to keep supporting the experiment in the app or we can remove it with the experimentation about this feature done.

Reply:
Got a confirmation on slack that it's safe to remove the experiment in 105 but this change should not be uplifted.